std_msgs/Header header
bool[] emergency
int16 task_state
float32 my_pos_x
float32 my_pos_y


